### ISSUE TITLE
`--createstub` with files too big

### ISSUE THREAD

**Issue by Avasam** [microsoft/pyright#4031]
Note: if you are reporting a wrong signature of a function or a class in the standard library, then the typeshed tracker is better suited for this report: https://github.com/python/typeshed/issues.

**Describe the bug**
Trying to generate stub for `cv2.cv2`.

**To Reproduce**
`pyright --createstub cv2`
or
`pyright --createstub cv2.cv2`

**Expected behavior**
Be able to generate stub. Maybe an optional flag to increase the max file length if `70563840` is too big for a sane default?

**Screenshots or Code**
Results:
```
Loading configuration file at C:\Users\Avasam\Documents\Git\typeshed\pyrightconfig.json
Assuming Python version 3.9
Assuming Python platform Windows
Auto-excluding **/node_modules
Auto-excluding **/__pycache__
Auto-excluding **/.*
File length of "c:\Users\Avasam\Documents\Git\typeshed\.venv\Lib\site-packages\cv2\cv2.pyd" is 70563840 which exceeds the maximum supported file size of 52428800
Type stub was created for 'cv2'
```

**VS Code extension or command-line**
CLI

**Comment by erictraut** [comment]
Stub generation won't work with cv2 because it's a binary library compiled from another language. Pyright's `--createstub` feature only works with Python source code. There are tools that can extract limited type information out of a binary through reflection, but it will be very low quality — not really usable for type checking.

**Comment by erictraut** [comment]
However, I agree that this error is incorrect. Pyright should still generate stubs for the parts of the library that are not found in binary files. I've fixed the cause of this issue, and partial stub creation now works for the cv2 library. This fix will be included in the next release.

**Comment by Avasam** [comment]
> Pyright's --createstub feature only works with Python source code.

Good to know! I'm using stubgen as well, which can get some info with introspection.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.275, which I just published. It will also be included in a future release of pylance.
